File: URI/python/1010.py
peca1 = input().split()
peca2 = input().split()

quantidadePeca1 = int(peca1[1])
quantidadePeca2 = int(peca2[1])
valorPeca1 = float(peca1[2])
valorPeca2 = float(peca2[2])

total = (quantidadePeca1 * valorPeca1) + (quantidadePeca2 * valorPeca2)

print("VALOR A PAGAR: R$ {:.2f}".format(total))

File: URI/Java/1010.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws IOException {
 
      Scanner sc = new Scanner(System.in);
      int c1, c2, n1, n2;
      double v1, v2, total;
      
      c1 = sc.nextInt();
      n1 = sc.nextInt();
      v1 = sc.nextDouble();
      c2 = sc.nextInt();
      n2 = sc.nextInt();
      v2 = sc.nextDouble();
      
      total = (n1 * v1) + (n2 * v2);
      System.out.format("VALOR A PAGAR: R$ %.2f\n", total);
    }
}
